tests/testthat/test-colspace.R

```diff
@@ -173,5 +173,17 @@ test_that("CIE", {
   expect_s3_class(implicit_cie_flowers, "colspace")
   explicit_cie_flowers <- colspace(vis_flowers, space = "ciexyz")
   expect_identical(implicit_cie_flowers, explicit_cie_flowers)
+  
+  # BYO data
+  fakedat <- data.frame(X = c(0.1, 0.2), 
+                        Y = c(0.3, 0.4),
+                        Z = c(0.5, 0.6),
+                        lum = c(NA, NA))
+  expect_equal(colspace(fakedat, space = "cielab", visual = sensdata(visual = 'cie10'), illum = sensdata(illum = 'D65')),
+               colspace(fakedat, space = "cielab", visual = 'cie10', illum = 'D65'))
+  
+  # Should ignore custom options when data are class vismodel()
```

Comment:
Should we add a warning in `cie()` to say these are ignored?

Reply:
I've added a more explicit note in the docs, but can't think of a clean way to do it atm, with the current setup. Will give it some more thought thought.